src/renderer/coremods/settings/pages/Addons.tsx

```diff
@@ -169,7 +169,7 @@ function Authors({ addon }: { addon: RepluggedPlugin | RepluggedTheme }) {
       ) : null}
       {author.github ? (
         <Tooltip
-          text="Open GitHub Profile"
+          text={Messages.REPLUGGED_ADDON_PROFILE_OPEN.format({ type: "GitHub" })}
```

Comment:
Should there be an i18n key for GitHub

Reply:
I honestly believe that "GitHub" should remain hardcoded. Discord also doesn't have an i18n key for it, so I'd say this is reasonable.